src/status_im/utils/config.cljs

```diff
@@ -20,10 +20,12 @@
 ;; flags stay up to date and are removed once behavior introduced is stable.
 
 (goog-define INFURA_TOKEN "800c641949d64d768a5070a1b0511938")
+(goog-define OPENSEA_API_KEY "")
```

Comment:
I think it makes sense to define it as `nil`. We can use any random value for this purpose but we already have `nil`.

Reply:
When i try to do:
```clj
(goog-define OPENSEA_API_KEY nil)
```
I just get:
```
IllegalArgumentException: no conversion to symbol
```
Which makes sense because:

>Use instead of [def](https://cljs.github.io/api/cljs.core/def) when you want to configure its value at compile-time with the [:closure-defines](https://cljs.github.io/api/compiler-options/closure-defines) compiler option. __Must be a string, number, or boolean.__

https://cljs.github.io/api/cljs.core/goog-define